architecture/core_concepts/routes.adoc

```diff
@@ -163,11 +163,26 @@ plug-in and finally into an HAProxy configuration:
 .HAProxy Router Data Flow
 image::router_model.png[HAProxy Router Data Flow]
 
-*Sticky Sessions*
+[[routes-sticky-sessions]]
+=== Sticky Sessions
 
-Implementing sticky sessions is up to the underlying router configuration. The
-default HAProxy template implements sticky sessions using the `balance source`
-directive which balances based on the source IP. In addition, the template
+Sticky sessions attempts to ensure that user traffic is directed to the same pod in each session, and improves application user experience by cacheing data. For example, for a cluster with five back-end pods, and two load-balanced routers, you can ensure that the same pod receives the web traffic from the same web browser regardless of the router that handles it.
```

Comment:
It feels like we need another word here, or maybe not. "The sticky sessions feature" or similar.

Is that how "cacheing" is spelled? Is is caching?
https://www.reddit.com/r/grammar/comments/472c97/caching_or_cacheing_in_the_context_of_computers/